scripts/docs/generate_asset_documentation.py

```diff
@@ -103,7 +103,8 @@ def create_asset_info(asset_config):
         if asset_config.type == AssetType.DATA:
             return DataInfo(asset_config)
         if asset_config.type == AssetType.PROMPT:
-            return PromptInfo(asset_config)
+            return None
+            # return PromptInfo(asset_config)
 
         raise Exception(f"Not supported asset type {asset_config.type}. Use {SUPPORTED_ASSET_TYPES}")
```

Comment:
You could instead check the asset type against SUPPORTED_ASSET_TYPES at the top of this function and warn (instead of raise exception) if it's not supported. I think that would better support benchmarking results.